tests/project-type/workspace-instance.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { WorkspaceImpl } from "@sap/project-api";
+import { expect } from "chai";
+import { uniq } from "lodash";
+
+import { mockVscode } from "../mockUtil";
+
+const testVscode = {
+    workspace: {
+        workspaceFolders: () => []
+    }
+};
+
+mockVscode(testVscode, "src/project-type/workspace-instance.ts");
+import { getBasWorkspaceAPI } from "../../src/project-type/workspace-instance";
+
+describe.skip("getBasWorkspaceAPI", () => {
+    it("multiple bas workspace instances should be equal", () => {
```

Comment:
"equal" --> "identical"

or "getBasWorkspaceAPI will always return the **same** instance"

Reply:
Done